app/move/controllers/preview-opt-in.js

```diff
@@ -1,14 +1,20 @@
+const { sendEvent } = require('../../../common/lib/analytics')
 const { COOKIES } = require('../../../config')
 const { PREVIEW_PREFIX } = require('../app/view/constants')
 
-function previewOptIn(req, res) {
+async function previewOptIn(req, res) {
   const moveId = req.query.move_id
   const cookieName = COOKIES.MOVE_DESIGN_PREVIEW.name(req.user.userId)
 
   res.cookie(cookieName, 1, {
     maxAge: COOKIES.MOVE_DESIGN_PREVIEW.maxAge,
   })
 
+  await sendEvent({
```

Comment:
It might be worth adding a `try/catch` around this with a call to `next(error)` if it fails.